.env.example

```diff
@@ -1 +1,6 @@
 REACT_APP_BACKEND_URL = https://tech-community.herokuapp.com/
+
+REACT_APP_CLOUD_NAME = tech-community
+REACT_APP_API_KEY = 478758755655457
```

Comment:
Variables should be empty and you don't need space before and after the `=` sign